x/stake/keeper/validator_test.go

```diff
@@ -93,6 +93,33 @@ func TestUpdateValidatorByPowerIndex(t *testing.T) {
 	require.True(t, keeper.validatorByPowerIndexExists(ctx, power))
 }
 
+func TestZeroPowerUnbonded(t *testing.T) {
+	ctx, _, keeper := CreateTestInput(t, false, 0)
```

Comment:
if the final term in `CreateTestInput` > 0 then I'm pretty sure loose tokens are created?/

Reply:
Does that matter for this test? Changed to `100` anyways.

Reply:
probably not, this is more in reference to 4 lines of code down where the loose bonded tokens are being set, I was suggesting maybe those lines could be deleted. Quite minor either way